nni/compression/pytorch/speedup/quantization_speedup/backend.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+class BaseModelSpeedup:
+    """
+    Base speedup class for backend engine
+    """
+    def __init__(self, model, config):
+        """
+        Parameters
+        ----------
+        model : pytorch model
+            The model to speed up by quantization.
+        config : dict
+            Config recording bit number and name of layers.
+        """
+        self.quantize_model = model
```

Comment:
it can simply be `self.model` instead of `self.quantize_model`, it should be the base model for both pruning and quantization

Reply:
Done.